charts/dgraph/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: dgraph
 version: 0.0.20
-appVersion: v21.12.0
+appVersion: v22.0.2
```

Comment:
we probably need 2 chart changes, one more for v23.0.0-beta1

Reply:
Helm chart is for production environments, so we don't advertise betas for production ready helm chart.  But this can be set with the `--set` helm install option.